langlib/lang.regexp/src/main/ballerina/regexp.bal

```diff
@@ -60,7 +60,7 @@ type GroupsArrayType GroupsAsSpanArrayType[];
 #
 # r.find("Hello World") is regexp:Span ⇒ true
 #
-# r.find("Hello World", 7) is regexp:Span ⇒ true
+# r.find("Hello World", 7) is regexp:Span ⇒ false
```

Comment:
Add we make this a true case and add another false case
```suggestion
# r.find("Hello World", 6) is regexp:Span ⇒ true
```

Reply:
Updated.